From afa3f6029bb6c232cde8cae80073b8ee0749b0bb Mon Sep 17 00:00:00 2001
From: Ferry Toth <ftoth@exalondelft.nl>
Date: Wed, 21 Dec 2022 22:39:23 +0100
Subject: [PATCH 1/1] [PATCH] u-boot: Fix path to merge_config.sh

Current u-boot recipe in kirkstone fails during do_configure because
it cannot find the merge_config.sh. Fix the path

Suggested-by: nishantpoorswani <nishantpoorswani@gmail.com>
Signed-off-by: Ferry Toth <ftoth@exalondelft.nl>
---
 meta/recipes-bsp/u-boot/u-boot-configure.inc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meta/recipes-bsp/u-boot/u-boot-configure.inc b/meta/recipes-bsp/u-boot/u-boot-configure.inc
index 04e0894752..195bac16a2 100644
--- a/meta/recipes-bsp/u-boot/u-boot-configure.inc
+++ b/meta/recipes-bsp/u-boot/u-boot-configure.inc
@@ -18,7 +18,7 @@ do_configure () {
                 if [ $j -eq $i ]; then
                     oe_runmake -C ${S} O=${B}/${config} ${config}
                     if [ -n "${@' '.join(find_cfgs(d))}" ]; then
-                        merge_config.sh -m -O ${B}/${config} ${B}/${config}/.config ${@" ".join(find_cfgs(d))}
+                        ${S}/scripts/kconfig/merge_config.sh -m -O ${B}/${config} ${B}/${config}/.config ${@" ".join(find_cfgs(d))}
                         oe_runmake -C ${S} O=${B}/${config} oldconfig
                     fi
                 fi
@@ -33,7 +33,7 @@ do_configure () {
         else
             oe_runmake -C ${S} O=${B} oldconfig
         fi
-        merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
+        ${S}/scripts/kconfig/merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
         cml1_do_configure
     fi
 }
-- 
2.37.2

